ecs-init/docker/dependencies.go

```diff
@@ -40,8 +46,15 @@ type _dockerclient struct {
 	docker *godocker.Client
 }
 
+func getDockerUnixSocket() string {
```

Comment:
Can you move this function (and associated constants) out to the `config` package?